src/main/resources/application.properties

```diff
@@ -8,4 +8,5 @@ management.endpoint.health.show-details=always
 spring.application.name=spring-boot-metrics
 management.metrics.tags.application=${spring.application.name}
 
-spring.profiles.active=dev
\ No newline at end of file
+jwt.secret=QlU+frI0AZIvyhzA9t+NDk9boRPbSJIEv9gBViCDc6XKgJZFqzw9w5JyGVRqspJXh7toUyFWP/XA0XVsCtqy9w==
```

Comment:
Please not that in prod code such data should not be stored here. Secrets and password must be kept in corporate credentials stores.